eng/generate-vssdk-versions.csx

```diff
@@ -45,7 +45,8 @@ foreach (var node in vssdkPackageSpec.Descendants())
         if (!id.StartsWith("Microsoft.VisualStudio") &&
             !id.StartsWith("Microsoft.ServiceHub") &&
             id != "Newtonsoft.Json" &&
-            id != "StreamJsonRpc")
+            id != "StreamJsonRpc" &&
+            id != "Nerdbank.Streams")
```

Comment:
I wouldn't overthink this right now. The script is run manually. If it doesn't generate version number that's needed it can always be easily updated as needed.